patches/react-native-screens+3.31.1.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenViewManager.kt b/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenViewManager.kt
+index c45adc0..338a571 100644
+--- a/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenViewManager.kt
++++ b/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenViewManager.kt
+@@ -42,10 +42,10 @@ open class ScreenViewManager : ViewGroupManager<Screen>(), RNSScreenManagerInter
+         props: ReactStylesDiffMap?,
+         stateWrapper: StateWrapper?
+     ): Any? {
+-        if (BuildConfig.IS_NEW_ARCHITECTURE_ENABLED) {
+-            // fabricViewStateManager should never be null in Fabric. The null check is only for Paper's empty impl.
+-            view.fabricViewStateManager?.setStateWrapper(stateWrapper)
+-        }
++//        if (BuildConfig.IS_NEW_ARCHITECTURE_ENABLED) {
++//            // fabricViewStateManager should never be null in Fabric. The null check is only for Paper's empty impl.
++//            view.fabricViewStateManager?.setStateWrapper(stateWrapper)
++//        }
```

Comment:
Upgrading screens to latest introduces a graphical bug which we do not want. However, this block fails on RN 0.76. It isn't useful anyway, since we are not using new arch yet (and the upgrade does not introduce the graphical bug in Fabric anyway, so we can upgrade when we go fabric)